packages/map-template/src/atoms/showLegendDialogState.js

```diff
@@ -0,0 +1,8 @@
+import { atom } from 'recoil';
+
+const showLegendDialogState = atom({
```

Comment:
Such booleans I prefer to prefix with "is" (like isMapReadyState, isLocationClickedState etc.).
Can we name this following that pattern? Something like isLegendDialogVisible?

Reply:
Good idea ✅

Reply:
🎉